crates/bevy_pbr/src/render/view_transformations.wgsl

```diff
@@ -0,0 +1,170 @@
+#define_import_path bevy_pbr::view_transformations
+#import bevy_pbr::mesh_view_bindings as view_bindings
+
+/// World space:
+/// +y is up
+
+/// View space:
+/// -z is forward, +x is right, +y is up
```

Comment:
Forward is which direction? From the camera position into the scene? From the camera position into the monitor?